.env.example

```diff
@@ -4,3 +4,5 @@ GITHUB_ANALYZER_PASSWORD=postgres
 
 # Must be set as the webhook secret on your github app
 GITHUB_ANALYZER_WEBHOOK_SECRET=fake-secret
+REDIS_URL=heroku_redis_url
+SIDEKIQ_THREADS=3
```

Comment:
Aren't we setting the concurrency twice? Here and then in the Procfile? 👀

Reply:
Not really, the worker dyno actually is not working, i should delete that dyno.